cmd/src/server.rs

```diff
@@ -66,8 +66,18 @@ use tikv_util::{
 /// Run a TiKV server. Returns when the server is shutdown by the user, in which
 /// case the server will be properly stopped.
 pub fn run_tikv(config: TiKvConfig) {
+    // Sets the global logger ASAP.
+    // It is okay to use the config w/o `validate()`,
+    // because `initial_logger()` handles various conditions.
+    // TODO: currently the logger config has to be provided
```

Comment:
It needs to be supported in this PR. It's not backward compatible.

Reply:
It is OK, the order of code executing are just like before except `tikv::log_tikv_info()` moving forward.

Reply:
In the past, logger can be configured via command line or configuration file. I think the "before" should be version 3.0 and older.

Reply:
Sorry, the comment was misleading, what that want to said was the logger's config can not get from PD like other config, but it still can be configured via command line or configuration file, I will fix the comment.